### ISSUE TITLE
Error with console in silx view

### ISSUE THREAD

**Issue by t20100** [silx-kit/silx#2982]
First reported here: https://github.com/silx-kit/silx/pull/2974#issuecomment-606631446

"""
I receive an error message when I try to open the console:

`ERROR:silx.gui.qt._qt:<class 'TypeError'> object() takes no parameters File "/home/pk/projects/silx/build/lib.linux-x86_64-3.6/silx/gui/plot/PlotWindow.py", line 392, in _toggleConsoleVisibility parent=self) `
"""

**Comment by t20100** [comment]
I didn't manage to reproduce this issue with both the tip of silx and v0.12.0 .
Which version of silx, python, qtconsole, pyqt5 or pyside2 do you use?

**Comment by vallsv** [comment]
I noticed that `IPythonDockWidget` can be `None`. Can it be that?

**Comment by t20100** [comment]
Yes, from the code, if the IPython widget is not available, `IPythonDockWidget` is `None`, but in this case, the `_toggleConsoleVisibility` method is not used (i.e., not connected to a signal).

**Comment by vallsv** [comment]
Weird message, i don't really understand what it means, nor where it come from.

**Comment by t20100** [comment]
Me neither, it looks as if `IPythonDockWidget` constructor is that of `object`...

**Comment by payno** [comment]
If you were not able to reproduce 5 years ago and this was weird. Let's clean this one. Please reopen if you are able to reproduce.
